filebeat/tests/system/test_registrar.py

```diff
@@ -947,20 +947,15 @@ def test_clean_removed_with_clean_inactive(self):
         os.remove(testfile_path1)
 
         # Wait until states are removed from inputs
-        self.wait_until(
-            lambda: self.log_contains(
-                "Remove state for file as file removed"),
-            max_timeout=15)
+        self.wait_until(lambda: self.log_contains("Remove state for file as file removed"))
```

Comment:
@ruflin are we sure of removing the `max_timeouts`?

Reply:
yes, default is 10s. I think most of them are here for copy / paste / historical reasons. There are very few special cases where we need lower or higher value.